clients/react/src/styles/aim-offline.css

```
@@ -0,0 +1,59 @@
+
+.aim-offline-gate-card h1 {
+  margin: 0 0 0.75rem;
+  font-size: 1.5rem;
+}
+
+.aim-offline-pick {
+  margin-top: 1.25rem;
+  padding: 0.55rem 1.2rem;
+  font: inherit;
+  cursor: pointer;
+  border: 1px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)